modules/oaeservice/manifests/hilary.pp

```diff
@@ -40,6 +40,7 @@
   $web_domain = hiera('web_domain')
   $app_admin_tenant = hiera('app_admin_tenant', 'admin')
   $admin_domain = "${app_admin_tenant}.${web_domain}"
+  $guest_domain = "guest.${web_domain}"
```

Comment:
What about the guest tenant alias then? Currently it's `guest`.

A content item created in the guest tenant alias would have URL:

`/content/guest/abc123`

Should it be `network` instead?